escrow/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "escrow"
+version = "0.1.0"
+authors = ["Cardinal Cryptography"]
+edition = "2021"
+license = "Apache 2.0"
+
+[dependencies]
+ink_primitives = { version = "3.3", default-features = false }
+ink_metadata = { version = "3.3", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.3", default-features = false }
+ink_storage = { version = "3.3", default-features = false }
+ink_lang = { version = "3.3", default-features = false }
```

Comment:
I would make a use of `~` similarly to our other contract crates (unfortunately, ink dep versions often have a critical significance)

Reply:
I'll hook version to `~3.3.1`, as it is what it is currently resolved automatically by cargo now